apps/web/src/components/AccountSelectorModal/AccountSelectorPopover.tsx

```diff
@@ -0,0 +1,121 @@
+  type SocialAccount,
+} from "@umami/core";
+import { useSelectedNetwork } from "@umami/state";
+
+import { AccountInfoModal } from "./AccountInfoModal";
+import { RemoveAccountModal } from "./RemoveAccountModal";
+import { RenameAccountPage } from "./RenameAccountModal";
+import {
+  EditIcon,
+  ExternalLinkIcon,
+  SearchIcon,
+  ThreeDotsIcon,
+  TrashIcon,
+} from "../../assets/icons";
+import { useColor } from "../../styles/useColor";
+
+type AccountSelectorPopoverProps = {
+  account: Account;
+};
+
+export const AccountSelectorPopover = ({ account }: AccountSelectorPopoverProps) => {
+  const { openWith } = useDynamicDisclosureContext();
+  const color = useColor();
+  const network = useSelectedNetwork();
+
+  return (
+    <Popover variant="dropdown">
+      <PopoverTrigger>
+        <IconButton
+          alignSelf="flex-end"
+          color={color("500")}
+          background="transparent"
+          aria-label="Account actions"
+          icon={<ThreeDotsIcon />}
+          onClick={event => event.stopPropagation()}
+          size="xs"
+        />
+      </PopoverTrigger>
+      <Portal>
+        <PopoverContent maxWidth="204px">
+          <PopoverBody color={color("400")}>
+            <Box>
+              <Button
+                onClick={e => {
+                  e.stopPropagation();
+                  return openWith(<AccountInfoModal account={account} />);
+                }}
+                variant="dropdownOption"
+              >
+                <SearchIcon />
+                <Text color={color("900")} fontWeight="600">
+                  Account Info
+                </Text>
+              </Button>
+              <Button
+                onClick={e => {
+                  e.stopPropagation();
+                  return openWith(<RenameAccountPage account={account} />);
+                }}
+                variant="dropdownOption"
+              >
+                <EditIcon />
+                <Text color={color("900")} fontWeight="600">
+                  Rename
+                </Text>
+              </Button>
+              {account.type !== "mnemonic" && account.type !== "multisig" && (
+                <Button
+                  onClick={e => {
+                    e.stopPropagation();
+                    return openWith(
+                      <RemoveAccountModal
+                        account={account as SocialAccount | LedgerAccount | SecretKeyAccount}
+                      />
+                    );
+                  }}
+                  variant="dropdownOption"
+                >
+                  <TrashIcon />
+                  <Text color={color("900")} fontWeight="600">
+                    Remove
+                  </Text>
+                </Button>
+              )}
+              <Button
+                onClick={e => {
+                  e.stopPropagation();
+
+                  const url = `${network.tzktExplorerUrl}/${account.address.pkh}`;
+                  window.open(url, "_blank");
```

Comment:
this is a security issue https://docs.datadoghq.com/code_analysis/static_analysis_rules/jsx-react/jsx-no-target-blank/

please use a `<Link isExternal` instead. [Example](https://github.com/trilitech/umami-v2/pull/1650/files#diff-ada356f9b19ff984f99256292da9132e180abdb48ab4db214f6b084d9429b155R49)